README.md

```diff
@@ -53,7 +53,7 @@ configuration for the helper. If this is not the case you can modify
 PulseAudio sysconfig file (/etc/sysconfig/pulseaudio) and add following
 environment variables:
 
-    AUDIOSYSTEM_PASSTHROUGH_TYPE={qti,af}
+    AUDIOSYSTEM_PASSTHROUGH_TYPE={qti,af,hw2_0}
```

Comment:
It would be nice to have readme changes in separate commit. Also please document that hw2_0 is only dummy implementation for now.